modules/core/arrow-instances-core/src/main/kotlin/arrow/instances/try.kt

```diff
@@ -1,11 +1,56 @@
 package arrow.instances
 
 import arrow.Kind
-import arrow.core.*
```

Comment:
I'm not sure why my editor is making that change, but I can revert if you'd like?

Reply:
Sure, although it isn't a blocker.